packages/docusaurus/src/client/serverEntry.js

```diff
@@ -28,6 +28,23 @@ import {
 } from './LinksCollector';
 import ssrTemplate from './templates/ssr.html.template';
 
+// eslint-disable-next-line no-restricted-imports
+import {memoize} from 'lodash';
+
+const customEtaConfig = {
+  name: 'ssr-template',
```

Comment:
We can remove the `name` option: it's used only for Eta's built-in caching inside render() and compile(), which we're bypassing to render.